starlark/src/values/function.rs

```diff
@@ -136,7 +136,7 @@ pub type StarlarkFunctionPrototype =
     dyn Fn(&CallStack, TypeValues, Vec<FunctionArg>) -> ValueResult;
 
 pub struct Function {
-    function: Box<StarlarkFunctionPrototype>,
+    function: fn(&CallStack, TypeValues, Vec<FunctionArg>) -> ValueResult,
```

Comment:
Can you add somewhere a comment basically resaying why Function != Def (just putting the comment from the commit in the code)?

Reply:
We already have Function != WrappedMethod.

I should probably rename Function to NativeFunction instead, and move Function::new_def to Def::new and Function::new_self_call to WrappedMethod::new, and Function::new to NativeFunction::new.

What do you think?

Reply:
I'll add the comment and submit that as a separate PR.